packages-backend/server-express/README.md

```diff
@@ -0,0 +1,45 @@
+# @commercetools-backend/server-express
```

Comment:
Sorry, do we do `express` or `server-express`? Intuitively I would just go with `express` 🤷 😕

Reply:
Yeah I guess with the "backend" prefix we can just use `express`